packages/manager/modules/exchange/src/account/add/account-add.html

```diff
@@ -9,42 +9,45 @@
     >
         <oui-back-button
             data-on-click="$ctrl.messaging.resetMessages(); $ctrl.hide()"
-            ><span data-translate="exchange_account_add_back_button"></span
+        ><span data-translate="exchange_account_add_back_button"></span
         ></oui-back-button>
 
         <div
             class="mb-3"
             data-ng-bind=":: 'exchange_account_add_header' | translate"
         ></div>
 
-        <div
-            class="oui-field"
-            data-ng-if="$ctrl.accountCreationOptions.availableTypes.length > 1"
-        >
-            <label
-                class="oui-field__label oui-label"
-                for="accountType"
-                data-ng-bind=":: 'exchange_account_add_accountType_label' | translate"
-            ></label>
-            <div class="oui-field__control oui-field__control_m">
-                <label class="oui-select d-inline-block w-75">
-                    <select
-                        id="accountType"
-                        name="accountType"
-                        class="oui-select__input"
-                        data-ng-model="$ctrl.newAccount.accountType"
-                        data-ng-options="type.displayName for type in $ctrl.accountCreationOptions.availableTypes | orderBy:'displayName' track by type.displayName"
-                        required
-                        aria-describedby="accountTypeDescription"
-                    ></select>
-                    <span
-                        class="oui-icon oui-icon-chevron-down"
-                        aria-hidden="true"
-                    ></span>
-                </label>
-                <span
-                    data-ng-if="$ctrl.exchangeServiceInfrastructure.isDedicatedCluster()"
-                >
+        <!--Account info section-->
+        <div class="row oui-box oui-box_light border-0 p-0">
```

Comment:
**question:** Why removing borders and padding on `oui-box` element? Guideline implies to keep them for consistency.

Reply:
Innitialy i have used the box because it was give a good presentation with a title and a dedicated pargraphe.
But at the end i need only a styled title.

So i changed to  `oui-heading_4` class.